src/main/java/com/lpvs/service/LPVSGitHubService.java

```diff
@@ -252,21 +252,26 @@ public void commentResults(LPVSQueue webhookConfig, List<LPVSFile> scanResults,
                     lpvsDetectedLicenseRepository.saveAndFlush(detectedIssue);
                 }
                 commitCommentBuilder.append("</ul>");
+                if (null != webhookConfig.getHubLink()) {
+                    commitCommentBuilder.append("(");
+                    commitCommentBuilder.append(webhookConfig.getHubLink());
+                    commitCommentBuilder.append(")");
+                }
```

Comment:
What about removing [hubLink](https://github.com/Samsung/LPVS/blob/c67c4bf89294aa9926f052d25c44182528c07db0/src/main/java/com/lpvs/entity/LPVSQueue.java#L64) in LPVSQueue? I don't think that LPVS set and use the value.

Reply:
We prefer to keep this field in the table because it is used in the internal version of LPVS, and we want to use open source version internally.